lib/node_modules/@stdlib/stats/base/dstdevyc/docs/repl.txt

```diff
@@ -110,8 +107,7 @@
 
     // Using offset parameter:
     > var x = new {{alias:@stdlib/array/float64}}( [ 1.0, -2.0, 3.0, 2.0, 5.0, -1.0 ] );
-    > var N = {{alias:@stdlib/math/base/special/floor}}( x.length / 2 );
-    > {{alias}}.ndarray( N, 1, x, 2, 1 )
+    > {{alias}}.ndarray( 3, 1, x, 2, 1 )
```

Comment:
```suggestion
    > {{alias}}.ndarray( 3, 1.0, x, 2, 1 )
```

I should briefly state why we do this: namely, we use decimals to convey when an argument is expected to be integer-valued and when a value can have decimals. Here, the correction parameter need not be integer-valued. Hence, we add a decimal point. This is our way of using documentation to convey "type" information, even when everything in JS is a "double" from the Standards POV.